src/integrations/ga.js

```diff
@@ -1,9 +1,15 @@
+import { generate } from '../guids.js';
+
 let pollAttempt = 0;
 const maxPollAttempts = 60;
 
 function gaIntegration() {
     if (pollAttempt > maxPollAttempts) {
-        console.log('Evolv: unable to set uid');
+        console.log('Evolv: GA clientId not found, using generated UID');
```

Comment:
Perhaps `console.info()` or possibly `console.warn()` instead?

I try to avoid using `console.log()` in code if I can, reserving it for debugging. But if I must, I try to use one of the log methods with extra semantic meaning. Doing so also suggests the console message was deliberate. (I know this isn't the only occurrence of `console.log()` though 😉 )

Reply:
Changed to `console.warn` - thanks!